packages-exp/auth-exp/package.json

```diff
@@ -21,6 +21,8 @@
     "dev": "rollup -c -w",
     "test": "yarn type-check && run-p lint test:browser",
     "test:browser": "karma start --single-run",
+    "test:browser:unit": "karma start --single-run --unit",
+    "test:browser:integration": "karma start --single-run --integration",
     "test:browser:debug": "karma start",
```

Comment:
You probably want "--auto-watch" for the debug rule, which performs hot code reloading.

Reply:
Done